controller/agent.go

```diff
@@ -172,7 +175,33 @@ func (self *Controller) agentOpRaftRemove(m *channel.Message, ch channel.Channel
 	//	return err
 	//}
 	// _, err := c.WriteString("success\n")
-	handler_common.SendOpResult(m, ch, "raft.remove", "no yet implemented", false)
+
```

Comment:
Should be able to specify addr or id, shouldn't require addr.

Reply:
Updated, originally done that way to have join/leave have the same input structure

Reply:
Saying that makes zero sense here, that was mostly for the cli side which previously (before my change) required sending the address so I made it required here before too